src/__init__.py

```diff
@@ -17,3 +17,10 @@
     __all__.append('autoquality')
 except ImportError:
     pass
+
+
+try:
+    from src.integrations import clearml  # noqa: F401
```

Comment:
Please include test for this conditional import: you can use this test as a reference:
https://github.com/Toloka/toloka-kit/blob/main/tests/import_tests/test_autoquality.py

Also please create a separate environment in tox configuration and run this test separately:
https://github.com/Toloka/toloka-kit/blob/dfa2bb6b896670de9931ffebf978981a2c6e377a/tox.ini#L4
https://github.com/Toloka/toloka-kit/blob/dfa2bb6b896670de9931ffebf978981a2c6e377a/tox.ini#L33